package.json

```diff
@@ -43,6 +43,7 @@
     "prompts": "^2.3.0",
     "simple-git": "^1.126.0",
     "strip-ansi": "^6.0.0",
+    "ts-morph": "^7.0.0",
```

Comment:
Because it's easier to get module exports.

Can revisit in the future if we really want this dep or not. But so far I've come to trust it.